app/views/assignments/_assignment_date.html.erb

```diff
@@ -18,4 +18,8 @@
       (<%= distance_of_time_in_words_to_now(due_date) %>)
     <% end %>
   <% end %>
+  <% grace_credits_used = @current_role.grace_credits_used_for(assignment) %>
+  <% if grace_credits_used > 0 %>
+    <span> - Grace Credits Used: <%= grace_credits_used %></span>
```

Comment:
Move the string into a locale file (look at how other user-facing strings are defined in this file)

Reply:
Also, let's **bold** the number of grace credits used.